ddtrace/contrib/integration_registry/_registry_schema.json

```diff
@@ -22,17 +22,25 @@
         "type": "object",
         "description": "Min, max, and list of tested versions for a specific dependency.",
         "properties": {
-          "min": {
-            "description": "Minimum explicit version found in test lockfiles.",
+          "tested_min": {
+            "description": "Minimum version found in test lockfiles.",
             "type": "string"
           },
-          "max": {
-            "description": "Maximum explicit version found in test lockfiles.",
+          "tested_max": {
+            "description": "Maximum version found in test lockfiles.",
+            "type": "string"
+          },
+          "supported_min": {
```

Comment:
I wonder if this should be a Python version specifier, e.g. `>=5.3,<6`, since in most cases we probably just want `>=5`, and a few rare cases where we don't support the latest major version so we need to adjust and be explicit here in docs about that.

being a version specifier means we can use some of the packaging/version tooling to compare actual library versions against this specifier directly rather than needed to do the math ourselves on `Version(library_version) >= Version(supported_min) and Version(library_version) < Version(supported_max)`, since the comparisons/rules could get more complex (how can we specify something like `>=5,!5.3.4,<6.4`, everything between 5 and 6.4, but not 5.3.4 which has a known bug).

Reply:
I see two options (there's probably more but these come to mind):

1.   Have 1 version range specifier: `supported_range`
  a.   If we support latest, then the specifier is simple: `>=5.0.0` or whatever is the actual min
  b.   If we don't support latest, we add the top range to the specifier, ie: `>=5.0.0,<6` 
  c.   If we need to exclude certain versions, ie:, we do not work with versions `5.2` or `5.5`, we have an exclusion list: `excluded_support_ranges = [ '~5.2.0', `~5.5.0` ]`
  
 2. Have a min and max specifier, as the PR currently has. And also have that exclusion list. 
 
 Both are different sides of the same coin, I don't have much preference.